app/models/miq_request_workflow.rb

```diff
@@ -781,17 +768,27 @@ def set_default_values
   def set_default_user_info
     return if get_dialog(:requester).blank?
 
-    if get_value(@values[:owner_email]).blank?
-      unless @requester.email.blank?
-        @values[:owner_email] = @requester.email
-        retrieve_ldap if MiqLdap.using_ldap?
-      end
+    if get_value(@values[:owner_email]).blank? && @requester.email.present?
+      @values[:owner_email] = @requester.email
```

Comment:
No, the `:owner_email` dialog field is a text field which results in a string being stored in this key.